setup.py

```diff
@@ -186,6 +186,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'watchtower~=0.7.3',
 ]
 azure = [
+    'azure-batch==8.0.0',
```

Comment:
we should have >= here. By definition we should have an open-upper requirements in setup.py

Reply:
Alright. Noted. I'll change it. Thanks